redisearch/document.go

```diff
@@ -70,6 +71,12 @@ func (d Document) Set(name string, value interface{}) Document {
 	return d
 }
 
+// SetTTL sets the ttl in the document
+func (d Document) SetTTL(ttl int) Document {
```

Comment:
I can definitely see the value... But perhaps we should at least set a default TTL of -1 during the New.

Reply:
Without the set, is by default to -1

Reply:
Correct - I just prefer being explicit.

Reply:
I'm in holidays, I'll update this PR when I'm back

Reply:
Up